### DISCUSSION TITLE
macOS aarch64 packages all identify as Intel, not Apple Silicon

### DISCUSSION THREAD

**Discussion by ejfuture** [rancher-sandbox/rancher-desktop#5156] [Q&A] [answered]
After installing Rancher Desktop I noticed in Activity Monitor that it was identified as an Intel application. I checked the install DMG "Rancher.Desktop-1.9.1.aarch64.dmg" and it showed it also as Intel. I checked the other builds and all were Intel.

What am I missing?

Thanks in advance.

**Comment by jandubois** [comment, marked as answer]
The UI code is still Intel code using Rosetta; see https://github.com/rancher-sandbox/rancher-desktop/issues/1416#issuecomment-1028207957

**Reply by ejfuture** [reply]
Thank you. When is the estimated release to see this fully as native?

**Reply by jandubois** [reply]
Sorry, we don't have an estimated release date yet, but I've tentatively moved #1416 into the 1.11 milestone, to at least investigate if there are any reasons blocking us from doing a native build.

**Reply by ejfuture** [reply]
Thank you. Really appreciate the replies.
